avocado/core/main.py

```diff
@@ -51,13 +52,18 @@ def handle_exception(*exc_info):
     tmp, name = tempfile.mkstemp(".log", prefix, get_crash_dir())
     os.write(tmp, msg.encode('utf-8'))
     os.close(tmp)
-    # Print friendly message in console-like output
-    msg = ("Avocado crashed unexpectedly: %s\nYou can find details in %s\n"
-           % (exc_info[1], name))
+    if exc_info[0] == KeyboardInterrupt:
+        msg = "%s\nYou can find details in %s\n" % (exc_info[0].__doc__, name)
+        exit_code = exit_codes.AVOCADO_JOB_INTERRUPTED
+    else:
+        # Print friendly message in console-like output
+        msg = ("Avocado crashed unexpectedly: %s\nYou can find details in %s\n"
+               % (exc_info[1], name))
+        exit_code = exit_codes.AVOCADO_GENERIC_CRASH
     os.write(2, msg.encode('utf-8'))
     # This exit code is replicated from avocado/core/exit_codes.py and not
     # imported because we are dealing with import failures
```

Comment:
Hi @beraldoleal I don't know if we should import `avocado.core.exit_codes`, because of this comment. Are you sure that this is not a problem anymore? And if it's ok, can you please remove that comment.

Reply:
On that particular line, I don't think that is valid, because the exception on line 21 will catch this. So I think that is save to remove the comment on line 64. @clebergnu can you confirm this, please?

Reply:
> On that particular line, I don't think that is valid, because the exception on line 21 will catch this. So I think that is save to remove the comment on line 64. @clebergnu can you confirm this, please?

Yes, AFAICT you're right.  I think we can remove the comment.